plugins/@grouparoo/spec-helper/src/lib/factories/apiKey.ts

```diff
@@ -1,5 +1,6 @@
 import { loadPath } from "../loadPath";
 import faker from "faker";
+import { ApiKey } from "@grouparoo/core";
```

Comment:
Note we import the Type form "@grouparoo/core", which needs to be a static import